RELEASES.md

```diff
@@ -10,14 +10,17 @@ for a complete commit history.
 [#xxxx](https://github.com/open-source-economics/Tax-Calculator/pull/xxxx))
 
 **API Changes**
-- Remove calculation of AGI tables from the TaxBrain Interface, tbi
-  [[#1724](https://github.com/open-source-economics/Tax-Calculator/pull/1724)
-  by Martin Holmer as suggested by Matt Jensen and Hank Doupe]
+- Change arguments for the three create-table utility functions and add easier to use Calculator methods that generate results tables
+  [[#1718](https://github.com/open-source-economics/Tax-Calculator/pull/1718)
+  by Martin Holmer]
 
 **New Features**
 - Add new policy parameter for refunding the new CTC against all payroll taxes
   [[#1716](https://github.com/open-source-economics/Tax-Calculator/pull/1716)
   by Matt Jensen as suggested by Ernie Tedeschi]
+- Remove calculation of AGI tables from the TaxBrain Interface, tbi
```

Comment:
@martinholmer, what made you switch this to a new feature from an API change? 

cc @hdoupe

Reply:
Dropping the AGI tables doesn't change the way you call any of the TBI functions, which is the most common cause of an API change.  If TaxBrain had been using (that is, displaying) the AGI tables, then I would have called #1718 an API change.  But TaxBrain 1.2.0 isn't doing anything with the AGI tables as far as I can see, so dropping them has no effect on TaxBrain.  That was the thinking.  If you would like to have it shown as an API change, just let me know and I'll move its position in the `RELEASES.md` file.

Reply:
That makes sense. Thanks. Fine as a feature enhancement.